pkg/plugins/handlers.go

```diff
@@ -76,6 +76,19 @@ func (p *PluginsHandler) HandlePlugins(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (p *PluginsHandler) HandleListPlugins(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		w.Header().Set("Allow", "GET")
+		serverutils.SendResponse(w, http.StatusMethodNotAllowed, serverutils.ApiError{Err: "Method unsupported, the only supported methods is GET"})
+		return
+	}
+	pluginsList := make([]string, 0, len(p.PluginsEndpointMap))
+	for k := range p.PluginsEndpointMap {
+		pluginsList = append(pluginsList, k)
+	}
+	serverutils.SendResponse(w, http.StatusOK, pluginsList)
+}
```

Comment:
Can you make this more general and also include the console version in the response? That way we can also prompt to refresh when console itself rolls out a new version.

I believe you can get the version from env var `SOURCE_GIT_COMMIT` when console is running in-cluster.

Reply:
@spadgett can you be more specific around what you'd like to see WRT making this more general? I've added the console version to the response in the form of a struct:

```go
struct {
    ConsoleVersion string   `json:"consoleVersion"`
    Plugins        []string `json:"plugins"`
}{
    ConsoleVersion: os.Getenv("SOURCE_GIT_COMMIT"),
    Plugins:        pluginsList,
}
```

Reply:
Looks good. We might rename the endpoint since it does more than just list plugins now, but that's not a critical issue.

I've confirmed that `SOURCE_GIT_COMMIT` does in fact track the openshift/console commit sha in product builds, so this looks like the right env var to use.

Reply:
Chatted with Sam this morning. We are going to rename `consoleVersion` to `consoleCommit` for clarity. So our new struct will look like:
```go
struct {
    ConsoleCommit string   `json:"consoleCommit"`
    Plugins        []string `json:"plugins"`
}{
    ConsoleCommit: os.Getenv("SOURCE_GIT_COMMIT"),
    Plugins:        pluginsList,
}
```